README.md

```diff
@@ -93,7 +93,7 @@ For environments where SSH keys are not feasible:
    sudo yum install sshpass
    ```
 
-2. **Configure MCP environment variables**:
+2. (optional) **Configure MCP environment variables**:
```

Comment:
This will still be required at the moment for simple-remote-launcher.js script to function unless we extend this functionality this script as well